core/server/models/base/index.js

```diff
@@ -133,12 +133,12 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
 
     // Get the user from the options object
     contextUser: function contextUser(options) {
-        // Default to context user
         if (options.context && options.context.user) {
             return options.context.user;
-        // Other wise use the internal override
         } else if (options.context && options.context.internal) {
             return 1;
+        } else if (options.context && options.context.client) {
```

Comment:
will wait that subscribe branch is merged into master, then i will simply delete this condition.